src/thor/timedep_forward.cc

```diff
@@ -104,7 +104,8 @@ bool TimeDepForward::ExpandForward(GraphReader& graphreader,
       // if this is a downward transition (ups are always allowed) AND we are no longer allowed OR
       // we cant get the tile at that level (local extracts could have this problem) THEN bail
       graph_tile_ptr trans_tile = nullptr;
-      if ((!trans->up() && hierarchy_limits_[trans->endnode().level()].StopExpanding()) ||
+      if ((!trans->up() &&
+           hierarchy_limits_[trans->endnode().level()].StopExpanding(pred.distance())) ||
```

Comment:
in the previous PR i accidentally removed the distance argument :facepalm:

Reply:
https://github.com/valhalla/valhalla/pull/2743/files#diff-1d57d4cfc830f3baf54a6c0378f8fb3c54aefcd414483503b2a02d8da591c280L115